ajax/libs/roundSlider/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "roundSlider",
+  "filename": "roundslider.min.js",
+  "description": "A free jQuery plugin",
+  "homepage": "http://roundsliderui.com/",
+  "keywords": [
+    "slider",
+    "round slider",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/soundar24/roundSlider"
+  },
+  "license": "MIT",
+  "author": "Soundara Pandian <@soundar24>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/soundar24/roundSlider.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "roundslider*"
```

Comment:
Although this rule work, we want all the files under `dist` folder.
So can you change to `**/*`?